internal/api/mfa.go

```diff
@@ -63,6 +64,7 @@ func (a *API) EnrollFactor(w http.ResponseWriter, r *http.Request) error {
 	user := getUser(ctx)
 	session := getSession(ctx)
 	config := a.config
+	db := a.db.WithContext(ctx)
```

Comment:
Guessing this is to help us add additional information to the traces for observability